### PR TITLE
Rahul mahendru

### PR DESCRIPTION
**Pull request by rahulmahendru**
Build Setup

### CODE REVIEW

**Comment by nahtnam** [comment]
Edit the `.gitignore` to include `build/` and also `Flowrite.app.zip`.

Then you need to run `git rm -r build` and `git rm Flowrite.app.zip`. Then commit and push

File: package.json [hunk 1]
```diff
@@ -45,12 +66,16 @@
     "concurrently": "^5.0.0",
     "electron": "^6.0.12",
     "electron-builder": "^21.2.0",
+<<<<<<< HEAD
```
Comment (nahtnam) [inline package.json]: Remove these weird lines (line 69, 71, 78)

**Review by nahtnam — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 2]
```diff
@@ -8,9 +8,6 @@
 # testing
 /coverage
 
-# production
-/build
```
Comment (nahtnam) [inline .gitignore:12]: Why delete this?

**Review by nahtnam — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (rahulmahendru) [reply .gitignore:12]: Resolved, deleted to push build, now it works! added it back

**Review by nahtnam — APPROVED** [review]

**Comment by rahulmahendru** [comment]
Updates Merged

**Pull request merged**